src/main/java/de/tum/in/www1/artemis/config/SecurityConfiguration.java

```diff
@@ -129,7 +130,7 @@ public void configure(WebSecurity web) {
         web.ignoring()
             .antMatchers(HttpMethod.GET, SYSTEM_NOTIFICATIONS_RESOURCE_PATH_ACTIVE_API_PATH);
         web.ignoring()
-            .antMatchers(HttpMethod.POST, ATHENE_RESULT_API_PATH + "*");
+            .antMatchers(HttpMethod.POST, "/api/athene-result/*");
```

Comment:
In a discussion with @krusche, we mentioned that the thing with constants is that it's really bad for searchability. Hence, I decided to replace them in my PRs, when I'm working on a part containing constants so this will get better over time.

I replaced the other ones.